### DISCUSSION TITLE
[Feature] Create album based on date span and original path

### DISCUSSION THREAD

**Discussion by plans-coding** [immich-app/immich#13616] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Normally, when I get back home from a trip, I like to add all the photos from that trip to an album. It would be a very nice feature if I could create an album based on:

* A date range (e.g., all photos between 2024-10-01 10:32 and 2024-10-05 15:45)
* The original folder path (only images from a specific folder, as others may add photos to different folders that were not part of the trip)

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
This will be covered by #1673

**Reply by bo0tzz** [reply]
Note that for now you can put your date range into a search filter and then create the album from there.

**Reply by plans-coding** [reply]
@bo0tzz It is not possible with originalPath in search filter, right?

**Reply by bo0tzz** [reply]
Indeed, that's not supported in the search filters yet.
